x-pack/plugins/maps/public/actions/ui_actions.ts

```diff
@@ -126,15 +126,15 @@ export function closeTimeslider() {
   };
 }
 
-export function setDeletedFeatures(layerId: string) {
+export function pushDeletedFeatureId(featureId: string) {
   return {
-    type: SET_DELETED_FEATURES,
-    layerId,
+    type: PUSH_DELETED_FEATURE_ID,
+    layerId: featureId,
```

Comment:
Please rename `layerId` to `featureId`. Little name flips like this make tracking things down the future much harder to follow.